fgpyo/read_structure.py

```diff
@@ -1,4 +1,66 @@
-# TODO: docstring with examples
+"""
```

Comment:
Doc-string is very helpful, maybe consider adding an example with ReadStructure.extract_with_quals()